setup/steps/autoexplain_recommended_settings.go

```diff
@@ -0,0 +1,519 @@
+		Message: fmt.Sprintf("Setting auto_explain.log_format is currently set to '%s'", currValue),
+		Help:    "Select EXPLAIN output format to be used (only 'text' and 'json' are supported)",
+		Options: optLabels,
+	}, &logFormatIdx)
+	if err != nil {
+		return "", err
+	}
+
+	if logFormatIdx == 0 || logFormatIdx == 1 {
+		return logFormatOpts[logFormatIdx], nil
+	} else if logFormatIdx == 2 {
+		return currValue, nil
+	} else {
+		panic(fmt.Sprintf("unexpected auto_explain.log_format selection: %d", logFormatIdx))
+	}
+}
+
+func getLogMinDurationValue(state *s.SetupState, currValue string) (string, error) {
+	if state.Inputs.Scripted {
+		if !state.Inputs.GUCS.AutoExplainLogMinDuration.Valid {
+			panic("auto_explain.log_min_duration setting needs review but was not provided")
+		}
+		return strconv.Itoa(int(state.Inputs.GUCS.AutoExplainLogMinDuration.Int64)), nil
+	}
+
+	var durationOpts = []string{
+		fmt.Sprintf("set to %dms (recommended inital value; will be saved to Postgres)", s.RecommendedGUCS.AutoExplainLogMinDuration.Int64),
+		"set to other value...",
+		fmt.Sprintf("leave at %sms", currValue),
+	}
+	var durationOptIdx int
+	err := survey.AskOne(&survey.Select{
+		Message: fmt.Sprintf("Setting auto_explain.log_min_duration is currently set to '%s ms'", currValue),
+		Help:    fmt.Sprintf("Threshold to log EXPLAIN plans, in ms; recommend %d, must be at least 10", s.RecommendedGUCS.AutoExplainLogMinDuration.Int64),
+		Options: durationOpts,
+	}, &durationOptIdx)
+	if err != nil {
+		return "", err
+	}
+
+	if durationOptIdx == 0 {
+		return strconv.Itoa(int(s.RecommendedGUCS.AutoExplainLogMinDuration.Int64)), nil
+	} else if durationOptIdx == 1 {
+		var logMinDuration string
+		err = survey.AskOne(&survey.Input{
+			Message: "Set auto_explain.log_min_duration, in milliseconds, to (will be saved to Postgres):",
+			Help:    "Threshold to log EXPLAIN plans, in ms; recommend 1000, must be at least 10",
+		}, &logMinDuration, survey.WithValidator(util.ValidateLogMinDurationStatement))
+		if err != nil {
+			return "", err
+		}
+		return logMinDuration, nil
+	} else if durationOptIdx == 2 {
+		return currValue, nil
+	} else {
+		panic(fmt.Sprintf("unexpected auto_explain.log_min_duration selection: %d", durationOptIdx))
+	}
+}
+
+func getLogNestedStatements(state *s.SetupState, currValue string) (string, error) {
+	if state.Inputs.Scripted {
+		if !state.Inputs.GUCS.AutoExplainLogNestedStatements.Valid {
+			panic("auto_explain.log_nested_statements setting needs review but was not provided")
+		}
+		return state.Inputs.GUCS.AutoExplainLogNestedStatements.String, nil
+	}
+
+	var logNestedIdx int
+	opts, optLabels := getBooleanOpts(currValue, s.RecommendedGUCS.AutoExplainLogNestedStatements.String)
+	err := survey.AskOne(&survey.Select{
+		Message: fmt.Sprintf("Setting auto_explain.log_nested_statements is currently set to '%s'", currValue),
+		Help:    "Consider statements executed inside functions for logging",
```

Comment:
Maybe just using this wording from the Postgres docs works a bit better:

`Causes nested statements (statements executed inside a function) to be considered for logging`